src/middlewares/isLogged.js

```diff
@@ -0,0 +1,34 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+const jwt = require('jsonwebtoken');
+
+const { SECRET_KEY } = process.env;
+
+// verify token cookie middleware
+const isLogged = (req, res, next) => {
+  const { token } = req.cookies;
```

Comment:
here the token name too
![image](https://user-images.githubusercontent.com/58285821/151672107-72ad57fe-94b9-4186-88b6-e9537873e827.png)